CODE_OF_CONDUCT.md

```diff
@@ -1 +1 @@
-All contributors and maintainers must abide by the [Pangeo Code of Conduct](https://github.com/pangeo-data/governance/blob/master/conduct/code_of_conduct.md).
+All contributors and maintainers must abide by the [xarray-contrib organization Code of Conduct](https://github.com/xarray-contrib/.github/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
One option would be to remove this file all together. If you do, GitHub will pick up the org-wide code of conduct and insert it into the UI.

Reply:
OK, let's do that. I don't think it's important for the code of conduct to be included in the distribution, since all xbatcher development is happening on GitHub.